example_connection.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+"""A simple pyisy test template."""
```

Comment:
Perhaps this file should go in to a folder like "examples" to make it clear that it's not a part of the module? I suppose it's already outside of the PyISY folder which does communicate that to some degree. What do you think?

Reply:
Works for me!